command.go

```diff
@@ -376,8 +376,15 @@ func (c *Command) VisibleCommands() []*Command {
 // VisibleFlagCategories returns a slice containing all the visible flag categories with the flags they contain
 func (c *Command) VisibleFlagCategories() []VisibleFlagCategory {
 	if c.flagCategories == nil {
-		c.flagCategories = newFlagCategoriesFromFlags(c.Flags)
+		c.flagCategories = newFlagCategories()
```

Comment:
dont we want to keep the newFlagCat..FromFlags() call ?

Reply:
👍🏼 this looks like a mis-merge to me. Thank you!